src/index.js

```diff
@@ -1,19 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import { GlobalStyle } from './GlobalStyle';
+import React from "react";
```

Comment:
Try to stay consistent with single or double quotes throughout your code base. Configure prettier in VS Code or add a .prettierrc file :)